server/services/offer.service.js

```diff
@@ -337,6 +365,14 @@ export const createOffer = async (offer) => {
     email: user.email,
   };
 
+  const status = offer.additionalClause
```

Comment:
I think this can be rewritten as:

```suggestion
  const status = offer.status || offer.additionalClause ? OFFER_STATUS.PENDING_ADMIN_APPROVAL : OFFER_STATUS.GENERATED
```

Reply:
Not in this scenario. This was written for test cases where `additionalClause` exists, and we want to use a custom status asides `PENDING_ADMIN_APPROVAL` and `GENERATED`